packages/tasks/src/model-libraries.ts

```diff
@@ -366,6 +366,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 			wildcard: { path: "*.sentis" },
 		},
 	},
+	"audiocraft-musicgen": {
```

Comment:
Libraries in this file are listed by alphabetical order. Would be good to keep the same (so moving this entry between `asteroid` and `bertopic`).